src/server/controllers/s3.js

```diff
@@ -22,8 +31,7 @@ module.exports = function s3 (config) {
   const client = new S3({
```

Comment:
I guess we can do `new AWS.S3` here now

Reply:
I wasn't quite sure about that. The documentation on how imports ought to work is a little awkward for aws-sdk and I personally tend to err on the side of utilizing aws-sdk/clients imported directly. That being said, this might just be voodoo. It ought to be verifiable by looking at the source.